.github/CODEOWNERS

```diff
@@ -1328,6 +1328,9 @@ src/platform/plugins/shared/discover/public/context_awareness/profile_providers/
 # Observability UI
 /x-pack/test_serverless/api_integration/test_suites/observability/config.ts @elastic/observability-ui @elastic/appex-qa
 /x-pack/test_serverless/api_integration/test_suites/observability/index.ts @elastic/observability-ui
+/x-pack/test_serverless/api_integration/test_suites/observability/config.logs_essentials.ts @elastic/observability-ui @elastic/appex-qa
+/x-pack/test_serverless/api_integration/test_suites/observability/index.logs_essentials.ts @elastic/observability-ui
+/x-pack/test_serverless/api_integration/test_suites/observability/logs_essentials_only @elastic/observability-ui
```

Comment:
Out of curiosity – What about ownership of the _functional_ FTR config? e.g. `x-pack/test_serverless/functional/test_suites/observability/config.logs_essentials.ts` I don't think QA is owner of that file, for example.

Reply:
Yes, ownership was growing organically a bit in this area and we're not perfectly consistent everywhere. I've noticed that as well, but considered it out of scope for this PR.